### DISCUSSION TITLE
Problema non errore del registro

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#708] [Richieste di aiuto] [answered]
Buonasera io quando vado a scrutininare una classe mi escono due opzioni: scrutinio secondo quadrimestre e scrutinio finale.
Lo scrutinio del secondo quadrimestre non serve ma mi serve solo quello finale come posso risolvere. Il cio è strano pk risultano solo 2 periodi cioè primo e secondo quadrimestre 
![Screenshot_2024-05-23-20-11-33-284_com android chrome-edit](https://github.com/iisgiua/giuaschool/assets/137722020/be4afd7d-5eb3-47ca-90f9-d8a53a0c9606)

**Comment by trinko** [comment]
Controlla i parametri in **Sistema>Parametri**: il parametro **periodo3_nome** probabilmente non è vuoto.

**Reply by Davide031** [reply]
E vuoto.

**Comment by trinko** [comment, marked as answer]
> E vuoto.

Allora è successo questo:
- tempo fai hai attivato i 3 periodi inserendo un valore nel parametro _periodo3_nome_;
- poi ha inserito la data dello scrutinio del secondo periodo dalla pagina **SCUOLA > SCRUTINI** (probabilmente per fare delle prove);
- successivamente sei tornato alla gestione a due periodi rimuovendo il valore presente nel parametro _periodo3_nome_.

Di conseguenza, pur usando ora i due periodi è rimasta l'impostazione con la data degli scrutini inserita per il secondo periodo.  

Per risolvere devi entrare in **PHPMyAdmin** dalla pagina di controllo del sito di Altervista, quindi cliccare sul pulsante SQL e inserire nel riquadro il comando seguente:
```
delete FROM gs_definizione_consiglio WHERE periodo='S'; 
```
Infine premi il pulsate di esecuzione.

**Comment by Davide031** [comment]
Ok grazie mille ha funzionato
